src/rqt_plot/plot.py

```diff
@@ -51,9 +51,10 @@ def __init__(self, context):
 
         self._context = context
         self._node = context.node
+        self._spinner = context.spinner
         self._args = self._parse_args(context.argv())
-        self._widget = PlotWidget(
-            self._node, initial_topics=self._args.topics, start_paused=self._args.start_paused)
+        self._widget = PlotWidget(self._node, self._spinner, initial_topics=self._args.topics,
```

Comment:
Nit: I'm partial towards putting args on a newline. It makes it easier to update the formatting if you change the method name.